Tutorial_1a_Convolving_Images.ipynb

```diff
@@ -0,0 +1,693 @@
+{
```

Comment:
<p>Suggested rewording and typos in "illustration" and "below", "We have additional date at 33" resolution to which we wish to compare our CO(2-1) observations, so we need to convolve the IRAM 30m data to the required resolution (see illustration below)." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>